vtrace/breakpoints.py

```diff
@@ -96,9 +99,13 @@ def resolveAddress(self, trace):
         if self.address is None and self.vte:
             try:
                 self.address = trace.parseExpression(self.vte)
+
             except Exception as e:
-                logger.warning('Failed to resolve breakpoint address for expression: %s', self.vte)
-                logger.warning('Error:', exc_info=1)
+                # this will happen with unresolved breakpoints.  
+                # depending on when resolution happens, the library may not have loaded yet.
+                if not self._complained:
+                    logger.warning('Failed to resolve breakpoint address for expression: %s (delayed resolution?)', self.vte)
+                    self._complained = True
```

Comment:
instead of complaining *every time* we run `resolveAddress` let's just complain once for each unresolved bp.